contracts/finance/payment-splitter/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "payment-splitter"
+version = "0.3.0-rc1"
+authors = ["Supercolony <green.baneling@supercolony.net>"]
+edition = "2018"
+
+[dependencies]
+ink_primitives = { git = "https://github.com/Supercolony-net/ink", branch = "3.0-rc5", default-features = false }
+ink_metadata = { git = "https://github.com/Supercolony-net/ink", branch = "3.0-rc5", default-features = false, features = ["derive"], optional = true }
+ink_env = { git = "https://github.com/Supercolony-net/ink", branch = "3.0-rc5", default-features = false }
+ink_storage = { git = "https://github.com/Supercolony-net/ink", branch = "3.0-rc5", default-features = false }
+ink_lang = { git = "https://github.com/Supercolony-net/ink", branch = "3.0-rc5", default-features = false }
+ink_prelude = { git = "https://github.com/Supercolony-net/ink", branch = "3.0-rc5", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "2.0", default-features = false, features = ["derive"] }
+scale-info = { version = "0.6", default-features = false, features = ["derive"], optional = true }
+
+strum_macros = "0.21.1"
+brush = { path = "../../../utils/brush", default-features = false }
+payment_splitter_derive = { path = "./derive" }
+
+[lib]
+name = "payment_splitter"
+path = "lib.rs"
+crate-type = [
+    "rlib",
```

Comment:
why isn't it cdylib?

Reply:
Because our modules can't be compiled directly like a contract, only like a dependency. So, we don't need `cdylib`